.env.local

```
@@ -0,0 +1,13 @@
+# Backend URL
+NEXT_PUBLIC_BACKEND_URL=http://localhost:5002
+
+# Auth
+AUTH_SECRET="LI7cH9JgbcWbMid18DZoQVX2SYEG2TiPl3gQDGo63Z4="
+AUTH_URL="http://localhost:3000"
+
+# Mapbox
+NEXT_PUBLIC_MABBOX_TOKEN="pk.eyJ1IjoibXVzZWdhbGEiLCJhIjoiY21hNmIxNGIxMHB3NzJxb2kwMmtocDFtbSJ9.U2lA3Fn90YAxuzboEPt1KQ"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)